api/python/cellxgene_census/tests/experimental/pp/test_hvg.py

```diff
@@ -117,7 +117,7 @@ def test_hvg_vs_scanpy(
     scanpy_hvg.index.name = "soma_joinid"
     scanpy_hvg.index = scanpy_hvg.index.astype(int)
     assert len(scanpy_hvg) == len(hvg)
-    assert all(scanpy_hvg.keys() == hvg.keys())
+    assert [k for k in scanpy_hvg.keys() if k != "gene_name"] == list(hvg.keys())
```

Comment:
more pythonic to do something like:

> assert set(scanpy_hvg.drop('gene_name').keys()) == set(hvg.keys())

this also removes the column order dependency that will trip up the above code at some point